docusaurus.config.ts

```diff
@@ -60,7 +60,7 @@ const config: Config = {
                     position: "left",
                     label: "Tutorial",
                 },
-                { to: "/blog", label: "Blog", position: "left" },
+                { to: "#", label: "Blog", position: "left" },
```

Comment:
Docusaurus detects broken links. Since we don't have `/blog` and other pages yet, it won't build successfully. Changed broken links to `#` for now.